wpilib/wpilib/safepwm.py

```diff
@@ -0,0 +1,42 @@
+# validated: 2017-10-07 EN e1195e8b9dab edu/wpi/first/wpilibj/SafePWM.java
```

Comment:
Why did you create this file?

Reply:
I thought I got rid of that....hmm

It came from when I merged